t_tests_classifier_accuracies.m

```diff
@@ -549,7 +549,7 @@
             
             [MCC_Results] = multcomp_fdr_bh(ANALYSIS.RES.p_ttest(na,:), 'alpha', ANALYSIS.pstats);
             ANALYSIS.RES.h_ttest(na, :) = MCC_Results.corrected_h;
-            ANALYSIS.RES.bky_crit_alpha(na) = MCC_Results.critical_alpha;
```

Comment:
This modification changes the name of a variable stored in the group results .mat file, denoting the critical alpha level resulting from the Benjamini-Hochberg FDR correction. It was accidentally labelled as the bky critical alpha (corresponding to the Benjamini-Krieger-Yekutieli FDR control method).

I don't think that we have a default MC correction, although we could set one in the future (for example the maximum statistic permutation test is a good strong FWER control method).